Environment_oM/Climate/SpaceTime.cs

```diff
@@ -33,31 +33,31 @@
 
 namespace BH.oM.Environment.Climate
 {
-    [Description("An Environment SpaceTime object used for defining locations in space and time for climate analysis")]
+    [Description("An environment object used to describe time for climate analysis")]
     public class SpaceTime : BHoMObject, IClimateObject
     {
-        [Description("An Environment Location object specifying the latitude, longitude and other location specifics of the SpaceTime object")]
+        [Description("The location for the climate analysis")]
         public Location Location { get; set; } = new Location();
 
-        [Description("The year of the time for the space time object")]
+        [Description("A number that represents the year for the space time object (e.g. 1970)")]
         public int Year { get; set; } = 1900;
 
-        [Description("The month of the time for the space time object")]
+        [Description("A number that represents the month of the year for the space time object (between 1 to 12, where 1 is January)")]
```

Comment:
Might suggest removing redundant info for all of these.

Something like:
```suggestion
        [Description("A number between 1 to 12, where 1 is January.")]
```

Reply:
agreed